client/reader/controller.js

```diff
@@ -35,27 +36,6 @@ function renderFeedError( context, next ) {
 	next();
 }
 
-export function legacyRedirects( context, next ) {
```

Comment:
To keep all `/read/*` redirects at one place I have moved the implementation to `setupReadRoutes` function